presto-main/src/test/java/com/facebook/presto/sql/planner/TestInterpretedPageProjectionFunction.java

```diff
@@ -162,17 +162,17 @@ public void testSymbolReference()
     {
         Symbol symbol = new Symbol("symbol");
         ImmutableMap<Symbol, Integer> symbolToInputMappings = ImmutableMap.of(symbol, 0);
-        assertProjection("symbol", true, symbolToInputMappings, ImmutableMap.of(symbol, BOOLEAN), 0, createBlock(BOOLEAN, true));
-        assertProjection("symbol", null, symbolToInputMappings, ImmutableMap.of(symbol, BOOLEAN), 0, createNullBlock(BOOLEAN));
+        assertProjection("symbol", true, symbolToInputMappings, new TypeProvider(ImmutableMap.of(symbol, BOOLEAN)), 0, createBlock(BOOLEAN, true));
```

Comment:
You could have a shortcut method for `new TypeProvider(ImmutableMap.of(symbol, xxxx))` to keep test lines shorter.